library/std/src/fs.rs

```diff
@@ -2400,13 +2400,8 @@ pub fn create_dir<P: AsRef<Path>>(path: P) -> io::Result<()> {
 ///
 /// # Errors
 ///
-/// This function will return an error in the following situations, but is not
-/// limited to just these cases:
-///
-/// * If any directory in the path specified by `path`
-/// does not already exist and it could not be created otherwise. The specific
-/// error conditions for when a directory is being created (after it is
-/// determined to not exist) are outlined by [`fs::create_dir`].
+/// The function will return an error if any directory specified in path does not does not exist and could not
+/// be created. There may be other error conditions; see [fs::create_dir] for specifics.
```

Comment:
```suggestion
/// be created. There may be other error conditions; see [`fs::create_dir`] for specifics.
```

And backticks around the linked item